.github/workflows/test.yml

```diff
@@ -61,6 +61,14 @@ jobs:
           key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
           restore-keys: ${{ runner.os }}-pnpm-store-
 
+      - name: Install wasm-pack
+        uses: jetli/wasm-pack-action@v0.4.0
+        with:
+          version: 'latest'
+
+      - name: Build Rust wasm deps
+        run: pnpm --filter @anonklub/merkle-tree compile && pnpm --filter @anonklub/spartan-ecdsa compile
+
```

Comment:
I think the `devPreinstall` hook makes this step now redundant.
```suggestion
```

Reply:
You were right.
After testing we need to manually run this otherwise the cache is hit. Because it only cares about node deps (pnpm lock file) and knows nothing about the changes that impact rust files...